src/pages/STG_PanelRD.page

```diff
@@ -61,6 +61,20 @@
                     </apex:outputPanel>
                 </div>
             </div>
+            <div class="slds-form-element" style="{!IF(!isRD2Enabled, 'display:none', 'display:block')}">
+                <apex:outputLabel value="{!$ObjectType.npe03__Recurring_Donations_Settings__c.Fields.Installment_Opportunity_Auto_Creation__c.Label}" for="IOAC" styleClass="slds-form-element__label" />
+                <div class="slds-form-element__control">
+                    <div class="slds-form-element__control">
+                        <apex:outputText value="{!installmentAutoCreationLabel}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
```

Comment:
Why not use `stgService.stgRD.InstallmentOppAutoCreateAction__c` field here instead of `installmentAutoCreationLabel`?

Reply:
The displayed version is retrieve from the 3 custom label and the one stored onto the NPSP is keep in the enum. I need a label to convert the enum version and the custom label version in case the user change the label/view it in a different language.